ci/deps/travis-38.yaml

```diff
@@ -4,13 +4,16 @@ channels:
   - conda-forge
 dependencies:
   - python=3.8.*
+
+  # tools
   - cython>=0.29.13
+  - pytest>=5.0.1
+  - pytest-xdist>=1.29.0  # The rest of the builds use >=1.21, and use pytest-mock
```

Comment:
@TomAugspurger is there any reason for this difference, now that things are standardized?

Reply:
I'm not aware of a reason for having this at >=1.29